Kitodo-API/src/test/java/org/kitodo/utils/GuardTest.java

```diff
@@ -112,21 +112,21 @@ void isNotNullShouldFailForNullObjekt() {
     }
 
     @Test
-    @Tag("isPositive(double)")
+    @Tag("isPositive_double")
```

Comment:
```suggestion
    @Tag("isPositiveDouble")
```

Reply:
Technically your changes are fine, thank you! I only suggest this cosmetic change here (and similarly for the other lines below) because using camel-case might fit better to the remaining tags.

Reply:
I chose the tag exactly as the function name is being tested. Here the camel case is part of the function name.

![Unbenannt](https://github.com/kitodo/kitodo-production/assets/3040657/945c3dc8-f1fb-48c1-af1e-54aaf5161d73)

Underscores are the arguments.

![Unbenannt2](https://github.com/kitodo/kitodo-production/assets/3040657/7c649f03-631f-46d6-98da-ba73bc6df70d)


Two different methods are actually being tested here, which are different for different purposes, only the function name is the same, but the parameters are different. 

![Unbenannt3](https://github.com/kitodo/kitodo-production/assets/3040657/4adc3232-74ec-4637-abe4-cfe6e3847222)

So I chose the name for the tage.

Reply:
Thanks for explaining. I was more focused on the test function names like `isPositiveDoubleShouldNotFailForPositiveValue` which don't use an underscore. The suggested change is not a must for me – the pull request is also fine without it.